programs/bpf_loader/src/lib.rs

```diff
@@ -1520,6 +1520,11 @@ mod tests {
         );
     }
 
+    fn truncate_data(account: &mut AccountSharedData, len: usize) {
```

Comment:
won't `truncate` be a method on either `AccountSharedData` as a required trait method on `WritableAccount` that `AccountSharedData` implements?